src/lib/loaders/loaders_with_authentication/tracked_entity.ts

```diff
@@ -11,10 +11,10 @@ import { map, find, extend } from "lodash"
  * @param {string} [entityKeyPath] an optional path to a nested entity
  */
 const trackedEntityLoaderFactory = (
-  dataLoader,
-  paramKey,
-  trackingKey,
-  entityKeyPath
+  dataLoader: any,
```

Comment:
Don’t we know the type of this?

Reply:
Eh, somewhat - after some exploration it's a few levels deep inside the dataloader factories with inferred typing most of the way. Don't think we'll gain too much by having the /exact/ type, so I've tightened it to a func that returns a promise 👍